src/frame/dune

```diff
@@ -0,0 +1,5 @@
+(library
+   (name Frame)
+   (public_name frame.lib)
+   (libraries Pastel str)
```

Comment:
can we use Re instead of str? there are examples elsewhere in the RN repo (pastel uses it for example)